src/audit/dangerous_triggers.rs

```diff
@@ -7,38 +7,40 @@ use crate::finding::{Confidence, Finding, Severity};
 use crate::models::Workflow;
 use crate::state::AuditState;
 
-pub(crate) struct DangerousTriggers {
-    pub(crate) _state: AuditState,
-}
+pub(crate) struct DangerousTriggers {}
```

Comment:
Nit: you can make this a ZST:

```suggestion
pub(crate) struct DangerousTriggers;
```

Reply:
Done here https://github.com/woodruffw/zizmor/pull/192/commits/ee0f5ef3bc7e05a5c3860e22d1346059c7430b8a